code_samples/fabric_kusto/fabric_kusto_vector_query.ipynb

```diff
@@ -0,0 +1,469 @@
+    "return_fields = [\"title\", \"category\", \"content\", \"similarity\"]\n",
+    "\n",
+    "results_df = search_kusto(client, \n",
+    "                       kusto_db=kdb_database_name, \n",
+    "                       query_vector=query_vector, \n",
+    "                       index_name=text_index_name, \n",
+    "                       vector_fields=[vector_field], \n",
+    "                       return_fields=return_fields,\n",
+    "                       k=5\n",
+    ")\n",
+    "\n",
+    "results_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Metadata filtering with vector search\n",
+    "\n",
+    "This demo shows how to apply metadata filtering (SQL - where, order by etc.) on top of vector search."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = 'find me a distributed system'\n",
+    "query_vector = azure_oai_client.embeddings.create(input = [query], model=aoai_embedding_deployed_model).data[0].embedding\n",
+    "\n",
+    "vector_field = \"content_vector\"\n",
+    "return_fields = [\"title\", \"category\", \"content\", \"similarity\"]\n",
+    "\n",
+    "filter = \"category == 'Compute'\"\n",
+    "\n",
+    "results_df = search_kusto(client, \n",
+    "                       kusto_db=kdb_database_name, \n",
+    "                       query_vector=query_vector, \n",
+    "                       index_name=text_index_name, \n",
+    "                       vector_fields=[vector_field], \n",
+    "                       return_fields=return_fields,\n",
+    "                       filter=filter\n",
+    "\n",
+    ")\n",
+    "\n",
+    "results_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Cross column vector search\n",
+    "\n",
+    "This demo shows how to apply vector search on multiple columns."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = 'find me a distributed system'\n",
+    "query_vector = azure_oai_client.embeddings.create(input = [query], model=aoai_embedding_deployed_model).data[0].embedding\n",
+    "\n",
+    "vector_fields = [\"title_vector\", \"content_vector\"]\n",
+    "return_fields = [\"title\", \"category\", \"content\", \"similarity\"]\n",
+    "\n",
+    "results_df = search_kusto(client, \n",
+    "                       kusto_db=kdb_database_name, \n",
+    "                       query_vector=query_vector, \n",
+    "                       index_name=text_index_name, \n",
+    "                       vector_fields=vector_fields, \n",
+    "                       return_fields=return_fields,\n",
+    "                       k=5\n",
+    ")\n",
+    "\n",
+    "results_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Hybrid search\n",
+    "\n",
+    "TODO"
```

Comment:
We should either remove this section or provide the TODO in this PR

Reply:
removed for now.